requirements.txt

```diff
@@ -17,8 +15,8 @@ cython
 git+https://github.com/googlei18n/cu2qu.git
 git+https://github.com/googlei18n/glyphs2ufo.git
 git+https://github.com/googlei18n/ufo2ft.git
-git+https://github.com/robofab-developers/robofab.git
```

Comment:
LOL

the problem is always https://github.com/robofab-developers/robofab/issues/52